pkgs/by-name/go/google-chrome/package.nix

```diff
@@ -50,7 +50,7 @@
   coreutils,
 
   # command line arguments which are always set e.g "--disable-gpu"
-  commandLineArgs ? "",
+  commandLineArgs ? "--simulate-outdated-no-au='Tue, 31 Dec 2099 23:59:59 GMT'",
```

Comment:
I'm open to this being non-configurable and moving to the derivation as well.
This will remove the following:
![image](https://github.com/user-attachments/assets/810def1c-1ba0-43d4-8e13-76b3304be301)

Reply:
Yeh I think maybe better to move into the derivation to avoid unexpected changes if people override the commandLineArga in the future :)

Reply:
Interestingly I've not seen the actual issue

Reply:
I'm on NixOS 24.05 but I keep google-chrome-stable at the release version.
You probably don't see if it you have it on unstable I'd imagine?